frontend/src/components/shared/Tooltip/styled-components.ts

```diff
@@ -0,0 +1,23 @@
+import styled from "@emotion/styled"
+import { transparentize } from "color2k"
+
+export const StyledTooltipContentWrapper = styled.div(({ theme }) => ({
+  boxSizing: "border-box",
+  maxWidth: `calc(${theme.sizes.contentMaxWidth} - 4rem)`,
+  maxHeight: "300px",
+  overflow: "auto",
+  padding: "14px 16px", // make vertical padding slightly smaller to account for line height
+
+  [`@media (max-width: ${theme.breakpoints.sm})`]: {
+    maxWidth: `calc(100% - 2rem)`,
+  },
+  a: {
+    color: theme.colors.blue50,
```

Comment:
There's a `theme.colors.linkText` derived color that we can use here (I believe it's used in `StreamlitMarkdown` already). It just uses a brighter blue on darker backgrounds.